.github/workflows/e2e-tests.yml

```diff
@@ -50,10 +50,15 @@ jobs:
           cd client
           npx playwright install
 
-      - name: Smoke tests (without annotations feature)
+      - name: Smoke tests
         run: |
           cd client && CURRENT_SHARD=${{ matrix.shardIndex }} TOTAL_SHARDS=${{ matrix.shardTotal }} make smoke-test
 
+      - name: User-created data persistence smoke tests
```

Comment:
Adds a dedicated data-persistence smoke test pass, gated to shardIndex==1 to keep CI time bounded. This targets /w (auth-enabled) flows.